ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/trace/chain-data/blocks.json

```diff
@@ -499,6 +499,32 @@
           "data": "0x60806040523373ffffffffffffffffffffffffffffffffffffffff16fffe"
         }
       ]
+    },
+    {
+      "number": "0x1D",
```

Comment:
If we are adding blocks for 1D and 1E we should have test traces for those blocks as well.

Reply:
I just added it. good catch